build/config.gypi

```diff
@@ -0,0 +1,361 @@
+# Do not edit. File was generated by node-gyp's "configure" step
```

Comment:
Is this file supposed to be checked into the repo? I would think everything in the `/build`  directory would be excluded